addons/artillerytables/functions/fnc_adjustFire.sqf

```diff
@@ -0,0 +1,41 @@
+#include "script_component.hpp"
+/*
+ * Author: LorenLuke
+ * Adjusts a target point north and east, and recalculates a solution in air based on atmospheric conditions
+ *
+ * Arguments:
+ * 0: Own Position ASL; 
+ * 1: Target Position ASL
+ * 2: adjustment to the East (negative is west)
```

Comment:
Please capitalize these everywhere. Adjustment should also include units in argument description.